### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by tmasternak**
This release targets NServiceBus 7

### CODE REVIEW

File: src/NServiceBus.Metrics.ServiceControl.Msmq/NServiceBus.Metrics.ServiceControl.Msmq.csproj [hunk 1]
```diff
@@ -10,9 +10,10 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="2.1.0-rc0002" />
-    <PackageReference Include="NServiceBus" Version="6.2.0" />
+    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="3.0.1" />
```
Comment (Scooletz) [inline src/NServiceBus.Metrics.ServiceControl.Msmq/NServiceBus.Metrics.ServiceControl.Msmq.csproj]: Sorry if this is a dummy question, but why do we need nuspec in this project? Why can't we specify wide ranges in here and the lowest from the range in tests?

Comment (ramonsmits) [reply src/NServiceBus.Metrics.ServiceControl.Msmq/NServiceBus.Metrics.ServiceControl.Msmq.csproj]: The nuspec should be a range, but the build itself must target a specific version right?

File: packaging/nuget/nservicebus.metrics.servicecontrol.msmq.nuspec [hunk 2]
```diff
@@ -13,8 +13,8 @@
     <copyright>$copyright$</copyright>
     <tags>$tags$</tags>
     <dependencies>
-      <dependency id="NServiceBus.Metrics.ServiceControl" version="[2.1.0-rc0002,3.0.0)" />
-      <dependency id="NServiceBus" version="[6.2,7)" />
+      <dependency id="NServiceBus.Metrics.ServiceControl" version="[3.0.1,4.0.0)" />
+      <dependency id="NServiceBus" version="[7.0.1,8)" />
```
Comment (Scooletz) [inline packaging/nuget/nservicebus.metrics.servicecontrol.msmq.nuspec]: Do we still need the nuspec file?

**Comment by ramonsmits** [comment]
[Merge strategy mentioned by @Scooletz to circumvent the conflicts:](https://github.com/Particular/PlatformDevelopment/blob/master/guidelines/releasing/gitflow.md#merge-release-into-master-advanced)

```cmd
git checkout master
git commit-tree "release-3.0.0^{tree}" -p release-3.0.0 -p master -m "Merge 'release-3.0.0' into 'master' using only contents from 'release-3.0.0'"
```

**Comment by ramonsmits** [comment]
Executing the `git commit-tree` statement didn't result in a local commit but I found a simpler alternative:

```cmd
git checkout release-3.0.0
git merge -s ours master
```

This results in the master branch to be merged, but that none of its changes are used. After pushing it doesn't show any conflicts anymore and can be merged via the GH PR.

**Comment by ramonsmits** [comment]
@bording please review the usage of Particular.Packaging

**Comment by bording** [comment]
@ramonsmits As discussed, I did a review pass on the entire repo and cleaned up things, and not just the packaging stuff.

You'll notice that I left placeholder text for the package description. You should replace that with a real description before shipping! 😄 

One other thing I noticed is that you're not strong-naming these assemblies. I attempted to add that, but you're also not strong-naming the `NServiceBus.Metrics.ServiceControl` assembly 😢 

I'd recommend fixing that ASAP.

**Comment by bording** [comment]
Also, it might make sense to backport the changes I've made here to your other support branches to get things cleaned up everywhere.

**Comment by ramonsmits** [comment]
@bording I've created issues regarding strong naming https://github.com/Particular/NServiceBus.Metrics.ServiceControl/issues/43 https://github.com/Particular/NServiceBus.Metrics.ServiceControl.Msmq/issues/12

**Pull request merged**